res/css/views/elements/_Field.scss

```diff
@@ -127,6 +127,7 @@ limitations under the License.
         transform 0.25s ease-out 0s,
         background-color 0.25s ease-out 0s;
     font-size: $font-10px;
+    line-height: normal;
```

Comment:
Reset `line-height` so it acts normal regardless of environment (`ContextMenu` has a very large `line-height`)